docs/Configuration.md

```diff
@@ -58,6 +58,9 @@ over this.
 
 ### polling_interval
 
+A positive decimal number. Max 9 and 6 digits in the integer and fraction
+components respectively.
```

Comment:
I'd prefer 0.001 over 0.0001 for lower limit. Millisecond is a commonly used unit. Or 0.000001 since a microsecond is the granularity that Time::HiRes speaks of.

I don't see how we get less to test by limiting the range. We'll just be checking the extremevalue and something just outside of that no matter if its 10 or 10 million.

Reply:
Testing if it works with 10.000.000 seconds can take 116 days. Testing 10 seconds takes 10 seconds.

Reply:
I am fine with 0.001 as the lower limit. I we get a request of a smaller value we can consider it then. I do not think we will get that. Anyone that wants to test that is probably capable of patching the code.

Reply:
And the code has been updated to `5 and 3 digits`.